unison-cli/src/Unison/Codebase/Editor/HandleInput/AuthLogin.hs

```diff
@@ -1,36 +1,14 @@
 module Unison.Codebase.Editor.HandleInput.AuthLogin (authLogin) where
 
 import Control.Monad.Reader
-import Network.URI (URIAuth (..), parseURI)
-import System.IO.Unsafe (unsafePerformIO)
 import Unison.Auth.OAuth (authenticateCodeserver)
 import Unison.Codebase.Editor.HandleInput.LoopState
 import Unison.Codebase.Editor.Output (Output (CredentialFailureMsg, Success))
-import Unison.Prelude
 import Unison.Share.Types
 import qualified UnliftIO
-import UnliftIO.Environment (lookupEnv)
 
--- | This is the URI where the share API is based.
-defaultShareURI :: CodeserverURI
```

Comment:
This moved to `Unison.Share.Codeserver` since it didn't really belong here anyways.